lib/ex_prosemirror/block/hard_break.ex

```diff
@@ -0,0 +1,30 @@
+defmodule ExProsemirror.Block.HardBreak do
+  @moduledoc ~S"""
+  Hard Break tag
+  """
+
+  use ExProsemirror.Schema
+  use ExProsemirror.Encoder.Json, type: :hard_break
+
+  import Ecto.Changeset
+
+  @doc false
+  embedded_schema do
+  end
+
+  @doc false
+  def changeset(struct_or_changeset, attrs \\ %{}) do
+    struct_or_changeset
+    |> cast(attrs, [])
+  end
+
+  defimpl ExProsemirror.Encoder.HTML do
+    import Phoenix.HTML.Tag, only: [tag: 1]
+
+    def encode(%{attrs: _attrs}, _opts) do
```

Comment:
This will raise a no clause matching error with attrs undefined (it's not in the schema)
```suggestion
    def encode(_struct, _opts) do
```